find_max.py

```diff
@@ -0,0 +1,16 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index):
+    max = None
+    if max_index == 0:
+        print("Array is empty")
```

Comment:
because max = None this if statement will return None, is that what we want to do?